src/test/java/edu/kis/powp/jobs2d/window/command/CommandManagerController.java

```diff
@@ -59,14 +60,18 @@ public void deleteObservers() {
 	}
 
 	@Override
-	public ICompoundCommand getCurrentCommand() {
-		return (ICompoundCommand) commandManager.getCurrentCommand();
+	public void runCommand() {
+		new SelectRunCurrentCommandOptionListener(driverManager).actionPerformed(null);
```

Comment:
no so nice :(. Creating an instance only tu use it passing null object. You should probably directly write a working code.

Reply:
I'm not sure we have added that method. Looking into actual, original CommandManagerController.java on head repository I see there is the same line 🤔 https://github.com/iis-powp-2021/powp_jobs2d_projects/blob/io1-dev/src/test/java/edu/kis/powp/jobs2d/window/command/CommandManagerController.java